src/utils/aotf.js

```diff
@@ -118,27 +119,30 @@ import { WorkflowState } from '@/model/WorkflowState.model'
 
 /**
  * Associates icons with mutations by name.
- * {mutation.name: svgIcon}
- */
-export const mutationIcons = {
-  '': mdiCog, // default fallback
-  broadcast: mdiBullhorn,
-  clean: mdiDelete,
-  editRuntime: mdiPlaylistEdit,
-  hold: mdiPauseCircleOutline, // to distinguish from pause
-  kill: mdiCloseCircle,
-  log: mdiFileDocumentOutline,
-  message: mdiEmail,
-  pause: mdiPause,
-  play: mdiPlay,
-  poll: mdiRefreshCircle,
-  release: mdiPlayCircleOutline, // to distinguish from play
-  reload: mdiReload,
-  remove: mdiMinusCircleOutline,
-  resume: mdiPlay,
-  set: mdiVectorPolylineEdit,
-  stop: mdiStop,
-  trigger: mdiCursorPointer
+ * @param {string} name - mutation name
+ * @returns {string} - icon svg path
+ */
+export function getMutationIcon (name) {
```

Comment:
This reads more plesantly IMO: Are the computational advantages too?

Reply:
The main advantage is it provides a proper way of having a default fallback icon.

In terms of performance, it shouldn't matter for this use, as it is only called a dozen times or so. In compiled languages, `switch` statements are usually equivalent to lookup tables. However I was curious and looked into JS, but can't find any definitive information. It looks like it may be entirely engine-dependent: https://jsperf.app/pekoba